dataRetriever/txpool/argShardedTxPool.go

```diff
@@ -47,5 +49,9 @@ func (args *ArgShardedTxPool) verify() error {
 		return fmt.Errorf("%w: NumberOfShards is not valid", dataRetriever.ErrCacheConfigInvalidSharding)
 	}
 
+	if args.TxCacheBoundsConfig.MaxNumBytesPerSenderUpperBound == 0 {
```

Comment:
I will add a TODO, to analyze whether this check is really needed or it's superfluous. `shardedTxPool` should not be aware of such a detail.